src/services/event-service.ts

```diff
@@ -166,10 +172,14 @@ export class EventService<E = any> {
     loggerHandler.info(`${this.name} start`);
 
     try {
-
-      for (const [networkAddress, {info, returnValues}] of Object.entries(await this._getEventsOfNetworks())) {
+      const entries = await this._getEventsOfNetworks();
+      
+      for (const [networkAddress, {info, returnValues}] of Object.entries(entries)) {
         await this.loadActorWithAddress(networkAddress);
-        await Promise.all(returnValues.map(event => blockProcessor(event, info)));
+
+        for (const event of returnValues) {
+          await blockProcessor(event, info, chainId!);
```

Comment:
_while `blockProcessor` does not need the `chainId` argument_ the correct argument to be passed would be the one inside `info` because we already have the information necessary related to the network inside that object